modules.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module 'webrtc-signal-http-heartbeat';
```

Comment:
sorry, why is this needed?

Reply:
Its a workaround for the fact that none of the webrtc-signal-http-foo deps are typescripted. I @-ed you in another comment for more context!